src/azure-cli/azure/cli/command_modules/synapse/operations/workspace.py

```diff
@@ -15,12 +16,19 @@ def list_workspaces(cmd, client, resource_group_name=None):
 
 
 def create_workspace(cmd, client, resource_group_name, workspace_name, storage_account, file_system,
-                     sql_admin_login_user, sql_admin_login_password, location=None, enable_managed_virtual_network=None,
-                     tags=None, no_wait=False):
+                     sql_admin_login_user, sql_admin_login_password, location=None, key_name="default", key_identifier=None, enable_managed_virtual_network=None,
+                     allowed_aad_tenant_ids=None, tags=None, no_wait=False):
     identity_type = "SystemAssigned"
     identity = ManagedIdentity(type=identity_type)
     account_url = "https://{}.dfs.{}".format(storage_account, cmd.cli_ctx.cloud.suffixes.storage_endpoint)
     default_data_lake_storage = DataLakeStorageAccountDetails(account_url=account_url, filesystem=file_system)
+    if str(key_identifier).endswith('/'):
```

Comment:
>key_identifier [](start = 11, length = 14)

If `key_identifier` is none, will logic here fail or not?

Reply:
str(None).endswith('/')  will return false.

Reply:
I think it can be removed. Do you think so?

Reply:
Agree to remove the check logic. It seems not quite necessary. Server side should determine whether to accept backslash.